modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -313,7 +313,7 @@
 			if (!interacting && Util.inBrowserView(el)) {
 				var form = el.closest('form');
 
-				var focusable = !el.is(':disabled') && !el.is(':hidden');
+				var focusable = !el.is(':disabled') && !el.is(':hidden') && !el.parents(":disabled").length;
```

Comment:
If you want to test if `length > 0`, please do so explicitly instead of relying on JS's automatic type juggling.